pyof/v0x01/controller2switch/packet_out.py

```diff
@@ -98,7 +98,7 @@ def unpack(self, buff, offset=0):
                 attribute = deepcopy(class_attribute)
                 if attribute_name == 'actions':
                     length = self.actions_len.value
-                    attribute.unpack(buff[begin:begin+length])
+                    attribute.unpack(buff[begin:begin + length])
```

Comment:
What pep8 relaxed rules say is that the spaces are optional. Not that "there should be no spaces".
I changed it anyway.